app/views/items/index.html.erb

```diff
@@ -6,7 +6,7 @@
 <% end %>
 <section class="grid-container">
   <% @items.each do |item| %>
-    <% if item.active? %>
+    <% if item.status == true %>
```

Comment:
Think .active? was a helper method that checked if an item was enabled or disabled - this might affect SimpleCov